### PR TITLE
Update customer-key-manage.md

### PR DESCRIPTION
**Pull request by sathana87**

### CODE REVIEW

File: microsoft-365/compliance/customer-key-manage.md [hunk 1]
```diff
@@ -132,7 +132,7 @@ To create a DEP, you need to remotely connect to SharePoint Online by using Wind
   
 1. On your local computer, using a work or school account that has global administrator permissions in your organization, [Connect to SharePoint Online PowerShell](/powershell/sharepoint/sharepoint-online/connect-sharepoint-online?preserve-view=true&view=sharepoint-ps).
 
-2. In the Microsoft SharePoint Online Management Shell, run the Register-SPODataEncryptionPolicy cmdlet as follows:
+2. In the Microsoft SharePoint Online Management Shell, for a premium/standard key vault run the Register-SPODataEncryptionPolicy cmdlet as follows:
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-manage.md:137]:
```suggestion
2. In the Microsoft SharePoint Online Management Shell, for either a premium or standard key vault, run the Register-SPODataEncryptionPolicy cmdlet as follows:
```

File: microsoft-365/compliance/customer-key-manage.md [hunk 2]
```diff
@@ -143,6 +143,10 @@ To create a DEP, you need to remotely connect to SharePoint Online by using Wind
    ```powershell
    Register-SPODataEncryptionPolicy -PrimaryKeyVaultName 'stageRG3vault' -PrimaryKeyName 'SPKey3' -PrimaryKeyVersion 'f635a23bd4a44b9996ff6aadd88d42ba' -SecondaryKeyVaultName 'stageRG5vault' -SecondaryKeyName 'SPKey5' -SecondaryKeyVersion '2b3e8f1d754f438dacdec1f0945f251a’
    ```
+For a managed HSM run the Register-SPODataEncryptionPolicy cmdlet as follows:
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-manage.md:148]:
```suggestion
For a managed HSM key vault, run the Register-SPODataEncryptionPolicy cmdlet as follows:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @sathana87, kindly commit the suggested changes if you agree with them. Many thanks.

**Comment by yogkumgit** [comment]
@sathana87 Please implement the suggested changes. Thanks

**Comment by yogkumgit** [comment]
@sathana87 Gentle reminder

File: microsoft-365/compliance/customer-key-manage.md (hunk 2 above)
Comment (KCCross) [reply microsoft-365/compliance/customer-key-manage.md:148]: I don't think Managed needs to be capped.

File: microsoft-365/compliance/customer-key-manage.md (hunk 1 above)
Comment (KCCross) [reply microsoft-365/compliance/customer-key-manage.md:137]: Premium and Standard don't need to be capped unless they're part of the entire brand name. Perhaps write the entire brand out?

**Comment by KCCross** [comment]
@JohanFreelancer9 and @sathana87 please review.

File: microsoft-365/compliance/customer-key-manage.md (hunk 1 above)
Comment (JohanFreelancer9) [reply microsoft-365/compliance/customer-key-manage.md:137]: Thanks, @KCCross, I've changed my suggestion accordingly.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@sathana87, kindly review/commit the suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@sathana87 Please implement the suggested changes, in the event of no response we may may have to close this PR. Thanks

**Comment by yogkumgit** [comment]
@sathana87 We appreciate any feedback that improves the content in the Microsoft docs but we haven't had any response from you so we are going to close this issue.  
Please feel free to re-open this issue if you see an improvement is required in the specific area of this documentation. We will certainly update this documentation with that valuable information. Thanks again, for taking out some time to open the issue. Appreciate and encourage you to do the same in future also.

**Pull request closed without merging**